realsense2_camera/CMakeLists.txt

```diff
@@ -117,6 +117,7 @@ find_package(tf2 REQUIRED)
 find_package(diagnostic_updater REQUIRED)
 
 find_package(realsense2 2.54.1)
+find_package(realsense2-gl 2.54.1)
```

Comment:
This means we will need to publish realsense2-gl as well right?
We need to think about if we need to make this optional somehow

Reply:
Have added a cmake variable `BUILD_ACCELERATE_WITH_GPU`.

Reply:
As discussed, can you check how we do it in the viewer?
Do we depend on realsense-gl and can disable/enable on runtime?

Reply:
> As discussed, can you check how we do it in the viewer? Do we depend on realsense-gl and can disable/enable on runtime?

Yes, I think we should release/publish realsense2-gl as well.
The viewer is depending on realsense2-gl, and you can choose this from the settings of the viewer, whether to run or not the GLSL.
@Arun-Prasad-V, did you test if user changes the parameter from 0 to 1 during runtime, while the build was without OpenGL ?

Reply:
We need to follow this guidelines same as for the SDK.
1. Users w/o GPU are not affected (meaning users w/o GPU can install and run the ROS node)
3. Users with GPU can decide if to leverage the GPU or not (launch file is sufficient)
4. We need to activate the internal formal conversion using GPU as well

Reply:
> @Arun-Prasad-V, did you test if user changes the parameter from 0 to 1 during runtime, while the build was without OpenGL ?

Right now, dynamic change is not supported. User must set this param during launch.
And, if the build was without OpenGL, then this param won't be exposed to user.